README.md

```diff
@@ -12,117 +13,108 @@ Generic network plugin (experimental) is designed to handle networking use cases
 - Integration with existing IPAM tools for migrating customers
 - Handle NIC's capabilities for acceleration (SRIOV/Offload/etc.)
 
-The overall design is _not_ assumed to be complete, because of ongoing work in the docker community with regards to the suitable APIs to interface with network extensions like this. Regardless, flexibility in the design has been taken into consideration to allow using a different state driver for key-value synchronization, or a different flavor of a soft-switch i.e. linux-bridge, MAC VLAN, or OpenvSwitch.
+The overall design is _not_ assumed to be complete, because of ongoing work in
+the docker community with regards to the suitable APIs to interface with
+network extensions like this. Regardless, flexibility in the design has been
+taken into consideration to allow using a different state driver for key-value
+synchronization, or a different flavor of a soft-switch i.e. linux-bridge, MAC
+VLAN, or OpenvSwitch.
 
-The ability to specify the intent succinctly is the primary goal of the design and thus some of the specified user interface will change, and in some cases functionality will be enhanced to accommodate the same. Design details and future work is captured in a [docs/design.md](https://github.com/contiv/netplugin/blob/master/docs/Design.md).
+The ability to specify the intent succinctly is the primary goal of the design
+and thus some of the specified user interface will change, and in some cases
+functionality will be enhanced to accommodate the same. Design details and
+future work is captured in a
+[docs/design.md](https://github.com/contiv/netplugin/blob/master/docs/Design.md).
 
-Please do not use this code in production, until code goes through more testing and few critical open issues are resolved.
+Please do not use this code in production, until code goes through more testing
+and few critical open issues are resolved.
 
-###Building and Testing
+### Getting Started
 
-**Note:** Vagrant 1.7.4 and VirtualBox 5.0+ are required to build and test netplugin.
-
-- Build:
-
-  `make build`
-
-   Note:
-   - building the project requires at least 1.4 Go Version. Instructions to install Go can be found at: https://golang.org/doc/install
-
-- Run unit-tests:
-
-  `make unit-test`
-
-- Run system-tests:
-
-  `make system-test`
-
-###Trying it out 
+This will provide you with a minimal experience of uploading the intent and
+seeing the netplugin system act on it. It will create a network on your host
+that lives behind an OVS bridge and has its own unique interfaces.
 
-The netplugin produces two binaries, a netplugin daemon and a netdcli tool to interact with it. The binaries can tried out in a vagrant environment, which can be setup as follows.
+#### Step 1: Clone the project:
 
-`make demo`
+```
+$ git clone https://github.com/contiv/netplugin
+$ cd netplugin; make build demo ssh
+```
 
-Note:
-- Make sure VirtualBox is installed
+#### Step 2: Inside the VM, boot `netmaster` and `netplugin`
```

Comment:
sorry I should have noticed this earlier. Do we need to bring them up explicitly? I thought you were going to incorporate this as part of make-demo. Or is that a separate PR? Even if it is, the doc could be updated now.